Mathlib/Order/Filter/Basic.lean

```diff
@@ -3346,6 +3347,10 @@ theorem Set.MapsTo.tendsto {α β} {s : Set α} {t : Set β} {f : α → β} (h
   Filter.tendsto_principal_principal.2 h
 #align set.maps_to.tendsto Set.MapsTo.tendsto
 
+theorem Filter.EventuallyEq.comp_fun {f g : β → γ} (h : f =ᶠ[l'] g)
```

Comment:
This already exists as [Filter.EventuallyEq.comp_tendsto](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Filter/Germ.html#Filter.EventuallyEq.comp_tendsto) (and I think the existing name is better, but that is debatable). It should absolutely be moved around here though